tests/utils/test_response.py

```diff
@@ -0,0 +1,43 @@
+import click
+
+from Babylon.utils.response import CommandResponse
+
+
+def test_response_empty():
+    """Testing command responses"""
+    ctx = click.Context(click.Command('cmd'))
+    with ctx:
+        CommandResponse()
+
+
+def test_response_fail():
+    """Testing command responses"""
+    ctx = click.Context(click.Command('cmd'))
+    with ctx:
+        response = CommandResponse.fail()
+        assert response.status_code == CommandResponse.STATUS_ERROR
+
+
+def test_response_dict():
+    """Testing command responses"""
+    ctx = click.Context(click.Command('cmd'))
+    with ctx:
+        response = CommandResponse(status_code=CommandResponse.STATUS_OK, data={"oups": True})
```

Comment:
Why do we need an existing content for `data` if we don't check it in the following test ?

Reply:
Fixed